### PR TITLE
translations: Research translations for Marathi

### PR DESCRIPTION
**Pull request by MadElf1337**

### CODE REVIEW

File: translations/Marathi/research/notes.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Zulip Administration
+
+# Translation - Marathi
+
+## Overview:-
```
Comment (roberthoenig) [inline translations/Marathi/research/notes.md]: Can you remove the :- after headings?

File: translations/Marathi/research/notes.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Zulip Administration
+
+# Translation - Marathi
```
Comment (roberthoenig) [inline translations/Marathi/research/notes.md]: There should only be one level-one heading (a level-one heading is a heading with one #).

File: translations/Marathi/research/notes.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Overview:-
+
+* Marathi is a commonly used language in the state of Maharashtra in India.
+
+* This language is one of the most diverse languages in India.
+
+* Various Websites & Social Networks have added this language to their interface.
+  (e.g. Google, Facebook, Twitter to name a few)
+
+* This language is more informal, but is used formally too.
```
Comment (roberthoenig) [inline translations/Marathi/research/notes.md]: Can you clairfy here what usage would be better for Zulip - formal or informal language? (What style do other internet platforms use?)

**Comment by MadElf1337** [comment]
@roberthoenig The Instagram interface is the same for both English & Marathi
Other errors I will correct

**Comment by roberthoenig** [comment]
@MadElf1337 sounds good! Regarding Instagram, you should take additional screenshots for another platform, and use them instead of Instagram (two times the same English interface demonstrates if a platform translates Marathi, but doesn't help translating terms).

**Comment by MadElf1337** [comment]
ok sure!!
@roberthoenig

**Comment by MadElf1337** [comment]
@roberthoenig It's Done!!
Can You Review The Task?

**Comment by roberthoenig** [comment]
Nice work, approving and merging.

**Pull request merged**